secure.go

```diff
@@ -58,14 +63,20 @@ type Options struct {
 	// CustomBrowserXssValue allows the X-XSS-Protection header value to be set with a custom value. This overrides the BrowserXssFilter option. Default is "".
 	CustomBrowserXssValue string
 	// ContentSecurityPolicy allows the Content-Security-Policy header value to be set with a custom value. Default is "".
+	// Passing a template string will replace `{{ . }}` with a dynamic nonce value for each request which can be later retrieved using the Nonce function.
```

Comment:
Using a template for this seems like overkill, when all you need is a single string replacement. Consider just defining a magic pattern (e.g. "$NONCE"), and using strings.Replace for this.